lib/ansiblelint/__main__.py

```diff
@@ -79,19 +79,31 @@ def choose_formatter_factory(
     return r
 
 
-def hint_about_skips(matches: List["MatchError"]):
-    """Display information about how to skip found rules."""
+def report_outcome(matches: List["MatchError"], options) -> int:
+    """Display information about how to skip found rules.
+
+    Returns exit code, 2 if erros were found, 0 when only warnings were found.
+    """
+    failure = False
     msg = """\
 You can skip specific rules by adding them to the skip_list section of your configuration file:
 ```yaml
 # .ansible-lint
-skip_list:
+warn_list:  # or 'skip_list' to silence them completly
```

Comment:
You resolved this, but the typo is still there.

Reply:
done now